kyuubi-server/src/main/scala/org/apache/kyuubi/operation/KyuubiOperationManager.scala

```diff
@@ -81,7 +81,8 @@ class KyuubiOperationManager private (name: String) extends OperationManager(nam
       className: String,
       batchConf: Map[String, String],
       batchArgs: Seq[String],
-      recoveryMetadata: Option[Metadata]): BatchJobSubmission = {
+      recoveryMetadata: Option[Metadata],
+      isCleanupResourceFile: Boolean): BatchJobSubmission = {
```

Comment:
how about get it from `KyuubiBatchSessionImpl`, because we need to check the option to determine whether check the resource path in `KyuubiBatchSessionImpl` as well.

Reply:
how about use a reserved key to present whether the resource file is uploaded.

Reply:
Changed to get the `isCleanupResourceFile` option from `KyuubiBatchSessionImpl`.